.template/variants/web/.stylelintrc

```diff
@@ -3,7 +3,6 @@
     "@nimblehq/stylelint-config-nimble"
   ],
   "rules": {
-    "no-eol-whitespace": true,
```

Comment:
for future reference: this rule is [deprecated](https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules) on Stylelint 15